packages/eas-cli/src/commands/build/run.ts

```diff
@@ -97,13 +98,19 @@ export default class Run extends EasCommand {
 
     const selectedPlatform = await resolvePlatformAsync(platform);
 
+    if (platform === 'ios' && process.platform !== 'darwin') {
+      Errors.error('You can only run iOS build on macOS devices', {
```

Comment:
Copy paste? You are not building here

Reply:
Not a copy-paste 😄 I can do:
```suggestion
      Errors.error('You can only use an iOS simulator to run apps on macOS devices', {
```
Is it okay?

Reply:
:+1: